models/user.go

```diff
@@ -8,3 +8,22 @@ type User struct {
 	Token        string `json:"token"`
 	TokenExpires int64  `json:"tokenExpires"`
 }
+
+type CreateUserRequest struct {
+	Name         string            `json:"name"`
+	Email        string            `json:"email"`
+	Password     string            `json:"password"`
+	Username     string            `json:"username"`
+	CustomFields map[string]string `json:"customFields,omitempty"`
+}
+
+type UpdateUserRequest struct {
+	UserID string `json:"userId"`
+	Data   struct {
+		Name         string            `json:"name"`
+		Email        string            `json:"email"`
+		Password     string            `json:"password"`
+		Username     string            `json:"username"`
+		CustomFields map[string]string `json:"customFields,omitempty"`
+	} `json:"data"`
```

Comment:
Wonder if since both Data and CreateUserRequest use the same fields.. could it use a shared struct?  Or even using the CreateUserRequest here?  🤔 

@graywolf336 what do you think?